### ISSUE TITLE
Newest update of Logitech G Hub doesn't work with Aurora

### ISSUE THREAD

**Issue by NeoDaemon** [Aurora-RGB/Aurora#172]
### Expected Behaviour
 Custom profiles loading.

### Actual Behaviour
 Custom profiles won't apply

### Reproduction steps
 Just updating to LGHUB 2023.7.448768. LGHUB just autoupdated to this version and aurora Doesn't apply the custom profiles anymore.

### Frequency
 Always

### Aurora Version:

v185
### Previously an Issue?
 nope until LgHub updated itself.

[2023-09-03 21.19.24.log](https://github.com/Aurora-RGB/Aurora/files/12507825/2023-09-03.21.19.24.log)

**Comment by crycx** [comment]
I have version 2023.7.448769 and have problem to get aurora to control rgb

**Comment by crycx** [comment]
Actually it works, if g hub is active and then you enable Logitech from Device Manager

**Comment by sym5** [comment]
Does not work for me, 
![Capture](https://github.com/Aurora-RGB/Aurora/assets/22459884/a86a2f65-d44f-4768-9d2b-90b02ec89a9d)

That's the output from ghub

![Capture2](https://github.com/Aurora-RGB/Aurora/assets/22459884/51559477-5503-4d03-886e-d11df332bfe8)

this is from Aurora device manager

**Comment by sym5** [comment]
my version of ghub is 2023.7.448769

**Comment by crycx** [comment]
Are you using the RGB.net version? I'm using the other one

**Comment by Aytackydln** [comment]
RGB.NET implementation is broken right now. It needs some researching before I can fix it and there's other task I'm working on right now

**Comment by sym5** [comment]
> Are you using the RGB.net version? I'm using the other one. And I meant by active, that the g hub window is open and then you open the aurora and press enabled.

Yes, i couldn't get the other 1 to run, when i clicked start it did nothing, however i did what you suggested , had ghub open , then clicked on enabled ,then start and it is now working , thankyou....

**Comment by sym5** [comment]
> RGB.NET implementation is broken right now. It needs some researching before I can fix it and there's other task I'm working on right now

ok , i have it disabled as of now.

**Comment by sym5** [comment]
one more thing, i used to use aurora back in the day before development stopped, mainly for the razor chroma plugin for supported games, ie overwatch keyboard lighting on my g910 , does the plugin still work, do i need to install the chroma wrapper still? TIA

**Comment by Aytackydln** [comment]
Yes chroma integration works but wrapper dll is no longer supported. You just need to install chroma itself and it should work

**Comment by NeoDaemon** [comment]
> Actually it works, if g hub is active and then you enable Logitech from Device Manager

That's true, if the LGHUB window is active and you press start logitech in Aurora actually works, but if you close LGHUB windows it also kills the profiles working again, also doesn't survive a reboot, gonna try other options in the meantime.

**Comment by Aytackydln** [comment]
Sorry for the really late fix but Logitech (RGB.NET) is usable now but Lightsync integration breaks it for now

**Comment by NeoDaemon** [comment]
Working! thank you!

**Comment by mattnoir** [comment]
> Sorry for the really late fix but Logitech (RGB.NET) is usable now but Lightsync integration breaks it for now

for me, G502 kinda broke and doesn't light up at all, even though detecting lights works properly

**Comment by Aytackydln** [comment]
G502 layout may need some changes then

**Comment by mattnoir** [comment]
> G502 layout may need some changes then

the thing is, it doesn't work even with openRGB rgb.net, so I really have no clue what could be wrong. I was able to set it up in previous versions, when keyboard didn't work at all,  but now it's been reversed. it's so close to perfection!

**Comment by Aytackydln** [comment]
@mattnoir according to layouts in Artemis G502 layout in Aurora has the correct leds, so it should work...

When you are using OpenRGB you can just remap the leds in "Remap Devices" tab in Aurora settings. And if you do, can you tell me the led names in there?

**Comment by mattnoir** [comment]
@Aytackydln I actually can't, it doesn't let me choose, but if I select "all logo" and it's set to peripheral_logo, it lights up
so I believe that yes, the layout should be correct even in logitech rgb.net,  but for some reason it's not working for me. maybe I am the problem here

**Comment by Aytackydln** [comment]
@mattnoir what do you mean by "it doesn't let me choose"?

Btw, that UI changed recently and you need to click the led (on layouts, above) you want to remap after clicking the led id button

**Comment by NeoDaemon** [comment]
Also cant set the G502 logo in a profile, If i test the led in the remapper settings it blinks, but doesn't light up in the profile.

**Comment by mattnoir** [comment]
> @mattnoir what do you mean by "it doesn't let me choose"?
> 
> Btw, that UI changed recently and you need to click the led (on layouts, above) you want to remap after clicking the led id button

that means I am just dumb and didn't realize I can choose it by clicking on the LED layout lol. OpenRGB now works almost flawlessly (I just kinda hate that I need 3 pieces of software to do one thing, plus occasional spasm happens, so I hope for logitech to work properly soon.

But as NeoDaemon said, when I click on "test" button, it blinks, but it doesn't light up as it should

**Comment by Aytackydln** [comment]
@mattnoir no, it's UI's fault 😅

Many people had that issue, especially previously it was a selection box people expected the same behaviour.

For mouse led.. I'm afraid I may have to update rgb.net libraries but Artemis had some problems with those versions recently and I'm working on a big change about device manager. That fix may take some long time
